packages/oats-runtime/src/make.ts

```diff
@@ -382,7 +386,10 @@ function isMagic(field: string) {
 
 export function makeObject<
   P extends { [key: string]: Maker<any, any> | { optional: Maker<any, any> } }
->(props: P, additionalProp?: any, comparisorOrder?: string[]) {
+>(props: P, additionalProp?: any, comparisorOrder?: string[], type?: ObjectType) {
+  const fromNetwork = fromNetworkMap(type);
```

Comment:
we construct and cache mapping info from the type on the first use of the maker.